webscalesqlclient/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
 cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
+SET(CMAKE_C_FLAGS "-Wno-maybe-unitialized ${CMAKE_C_FLAGS}")
```

Comment:
You are right @steelbrain :) good catch. Fixed.

https://github.com/hhvm/hhvm-third-party/commit/c018f6cc31a21f52b10389fc0ef146d665bb3010